drivers/vulkan/vulkan_context.cpp

```diff
@@ -2348,7 +2348,10 @@ Error VulkanContext::swap_buffers() {
 	submit_info.signalSemaphoreCount = 1;
 	submit_info.pSignalSemaphores = &draw_complete_semaphores[frame_index];
 	err = vkQueueSubmit(graphics_queue, 1, &submit_info, fences[frame_index]);
-	ERR_FAIL_COND_V(err, ERR_CANT_CREATE);
+
+	if (err != VK_SUCCESS) {
+		ERR_FAIL_COND_V_MSG(err, ERR_CANT_CREATE, "Err: " + String(string_VkResult(err)));
```

Comment:
I think we can make the error message a bit more explicit. This is fine for us debugging something we can reproduce, but here these are errors that users will see, try to understand, and eventually share with us. It's best if it's written to be somewhat explicit.